clar2wasm/src/words/bindings.rs

```diff
@@ -124,11 +126,15 @@ mod tests {
     #[test]
     fn validate_let() {
         // Reserved keyword
-        crosscheck("(let ((map 2)) (+ map map))", Err(()));
+        // TODO: change that assertion to validate the exact error thrown. Handle that when issue #421 is complete.
+        assert!(evaluate("(let ((map 2)) (+ map map))").is_err());
```

Comment:
We should still check that this errors in both implementations, maybe add a crosscheck_both_fail for this?

Reply:
This way we can add the TODO in this function, instead of having it everywhere.

Reply:
Good call. Added a `crosscheck_expect_failure` function.